.dockerignore

```diff
@@ -10,6 +10,8 @@
 !az.completion
 # Make an exception for the license file
 !LICENSE.txt
+# Make an exception for docker related files
+!docker/*
```

Comment:
Add this to mount the rpm file, for
```
# Exclude everything by default
*
```

Reply:
The name is confusing. We can consider either renaming it or adding comments to clarify its content.